packages/truffle/test/scenarios/javascript_testing/assertions.js

```diff
@@ -22,6 +19,10 @@ describe("Javascript testing", function () {
     config.mocha = {
       reporter: new Reporter(logger)
     };
+    await Server.start();
+  });
+  after(async function () {
+    await Server.stop();
```

Comment:
Good idea! I'm not exactly sure the order that multiple `before`s and `after`s are resolved.